components/observations/ObservationsListView.tsx

```diff
@@ -1,44 +1,44 @@
-import React, {useCallback, useEffect, useMemo, useState} from 'react';
```

Comment:
Does our prettier config do this on your machine? Yikes

Reply:
Ha, just reverting this now. This looks like my global prettier config getting opinionated.